package.json

```diff
@@ -73,15 +73,15 @@
   },
   "dependencies": {
     "@patternfly/patternfly": "2.8.3",
-    "@patternfly/react-charts": "4.5.2",
-    "@patternfly/react-core": "3.70.1",
-    "@patternfly/react-icons": "3.9.3",
-    "@patternfly/react-styles": "3.4.4",
-    "@patternfly/react-table": "2.14.11",
-    "@patternfly/react-tokens": "2.6.9",
+    "@patternfly/react-charts": "4.7.2",
+    "@patternfly/react-core": "3.80.3",
+    "@patternfly/react-icons": "3.10.14",
+    "@patternfly/react-styles": "3.5.7",
+    "@patternfly/react-tokens": "2.6.13",
     "@redhat-cloud-services/frontend-components": "0.0.7",
     "@redhat-cloud-services/frontend-components-utilities": "0.0.7",
     "axios": "^0.19.0",
+    "classnames": "^2.2.5",
```

Comment:
i thought it was odd too. Noticed this after the upgrade. It looks like `@redhat-cloud-services/frontend-components` now has `classnames` as a peerDependency, and it's not getting installed by default. It will error without it after upgrade...

Reply:
ugh, yeah it probably shouldn't be a `peerDependency` since it's actually being referenced. that's k we'll leave it in, we always seem to use it eventually

- https://github.com/RedHatInsights/frontend-components/search?q=classnames&unscoped_q=classnames